source/base/process_grid.cc

```diff
@@ -170,9 +170,8 @@ namespace Utilities
 
       // Create the communicator based on the group
       // Note that on most cores the communicator will be MPI_COMM_NULL.
-      // FIXME: switch to MPI_Comm_create_group for MPI-3 so that only processes within the to-be subgroup call this
```

Comment:
while you're here, could you clarify in the comment that it is the *resulting communicator* or *output communicator* that will be `MPI_COMM_NULL`?